### DISCUSSION TITLE
Faciliter la gestion de inscriptions aux formations,embarquements et forum

### DISCUSSION THREAD

**Discussion by AmelAlili** [betagouv/cap#3] [Besoin]
Pouvoir:
- s'inscrire aux formations à venir
- se désinscrire facilement
- se mettre en liste d'attente
- valider sa présence aux formations pour recevoir les supports

**Comment by jdauphant** [comment]
Proposition de solution : intégration de la gestion de des formations dans l'espace membre

**Comment by fannyblanc** [comment]
Est-ce qu'on pourrait aussi envisager une solution pour l'inscription au forum ?

**Reply by jdauphant** [reply]
j'ai ajouté dans le besoin
